packages/cascara/src/modules/DataCheckbox/DataCheckbox.doc.mdx

```diff
@@ -1,10 +1,77 @@
 ---
-title: DataCheckbox
+title: Data Checkbox
 propTable: DataCheckbox.js
 type: 'module'
 public: false
 ---
 
+A data module to display **Boolean** values, it renders a checkbox.
+
+## Using the module
+
+Additional to the props specified aside, please use the following property to configure your module:
+
+| property | type                | description                                             |
+| -------- | ------------------- | ------------------------------------------------------- |
+| module   | required **String** | specifies the type of module, in this case **checkbox** |
+
+> Adding this module to your **Cascara** **Table** or **Form** would look like this:
+
+```jsx
```

Comment:
I worry that these JSX files will become a lot to maintain having the Table here. This also obscures the point that these are also used in Form.

I think we should instead render the module itself only in these code examples. If we do it this way... there is a LOT of code we will have to update if we need to change something with Table or Form and we are showing things outside of the scope of the module itself.

If we _did_ want to show this at some point, I think it would make more sense to make a Table and/or Form sandbox just for Docs that has all of this boilerplate inside of it for showing these modules. But I am pretty sure that our lives are going to be a lot easier not maintaining that and instead just render the modules themselves.

Reply:
You are right @brianespinosa, let me change the examples to use the module provider

Reply:
@brianespinosa we are now using both, the `ModuleProvider` and `Table` playgrounds